### PR TITLE
Updates to omero.properties 

### PR DESCRIPTION
**Pull request by joshmoore**
See: https://trello.com/c/cATe4yap/306-5-1-omero-properties

### CODE REVIEW

**Comment by joshmoore** [comment]
@sbesson: any thoughts on getting https://github.com/openmicroscopy/openmicroscopy/blob/develop/etc/profiles/psql into the docs? Perhaps I just fake the few that are worthwhile and make sure that precedence favors the values under `profiles`.

**Comment by sbesson** [comment]
@joshmoore: heh. I was thinking about this question while having a first look at this PR earlier. Let me see what I can do tonight (I'll open a PR against your branch if things work out. Out of curiosity, can you remind me of the original reason for having a separate psql profile file as opposed to storing all values under `omero.properties`?

**Comment by joshmoore** [comment]
These are all the values that make sense for just Postgres. Values for another database would go in a separate file. However, like we did with `SqlAction`'s `default.properties`, the PG values could largely be considered the defaults. It's just something that's tricky to test.

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OME-5.1-merge-push#993](https://ci.openmicroscopy.org/job/OME-5.1-merge-push/993/). See the [console output](https://ci.openmicroscopy.org/job/OME-5.1-merge-push/993/consoleText) for more details.
Possible conflicts:
  - PR #3790 joshmoore '10661 dynamic params'
    - etc/omero.properties
  - PR #3798 joshmoore 'LDAP-support for group-owners'
    - etc/omero.properties
  - PR #3810 aleksandra-tarkowska 'adding login params for autologin'
    - etc/omero.properties

**Comment by joshmoore** [comment]
Excluding until other PRs are in, then I'll merge `origin/develop`. Since this is documentation, it could conceivably happen *slightly* later than the rest.

File: etc/omero.properties [hunk 1]
```diff
@@ -662,56 +724,58 @@ omero.ldap.group_filter=(objectClass=groupOfNames)
...
+# Without a prefix the "new_user_group" property
+# specifies the name of a single group which all
+# new users will be added to.
+#
+# Other new_user_group strings are prefixed with
+# ":x: and specify various lookups which should
+# take place to find one or more target groups
+# for the new user.
+#
+# :ou: uses the final organizational unit of a user's dn
```
Comment (sbesson) [inline etc/omero.properties]: This needs to be enclosed using backquotes cf https://github.com/openmicroscopy/ome-documentation/blob/develop/omero/sysadmins/server-ldap.txt#L124

**Comment by joshmoore** [comment]
Pushed.

**Comment by sbesson** [comment]
See https://www.openmicroscopy.org/site/support/omero5.1-staging/sysadmins/config.html. Merging.

**Comment by sbesson** [comment]
As a follow-up for 5.1.2, we should likely remove the warning about `omero.ldap.sync_on_login`.

**Comment by joshmoore** [comment]
Not yet. There will need to be further client work and testing before that can be removed.

**Pull request merged**